operators/Cargo.toml

```diff
@@ -21,7 +21,14 @@ schemars = "0.8.10"
 serde = "1.0.143"
 serde_json = "1.0.83"
 serde_yaml = "0.9.9"
-tokio = {version="1.20.1", features = ["macros", "rt-multi-thread", "time"]}
+
+# The tokio crate version should be compatible with the one used in actix-web.
+# The upgrade controller is deployed as a tokio task, and the actix-web must be able to
+# use the same runtime for its listener and workers.
+# E.g., the tokio version used to build actix-web 4.2.1 is tokio 1.13.1.
```

Comment:
we can remove this comment as this is generally true for all crates with shared deps.
Also the way we set dependencies with the default resolver you're not guaranteed to get this exact version btw, it will be the "best for all"